docs/config.rst

```diff
@@ -525,3 +525,27 @@ In addition to these configurations:
   * Importance: low
 
 
+Interceptor Configuration Options
+==============================
+REST Proxy supports interceptor configurations as part of Java new producer and consumer settings.
+
+``producer.interceptor.classes``
+  Producer interceptor classes.
+
+  * Type: string
+  * Default: ""
+  * Importance: low
+
+``consumer.interceptor.classes``
+  Consumer interceptor classes.
+
+  * Type: string
+  * Default: ""
+  * Importance: low
+    
+For example to enable Confluent Control Center monitoring interceptors:
+
+``consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor``
```

Comment:
Could we add these as commented out examples in `config/kafka-rest.properties` as well so it's easy for people to add them there? Maybe with a small comment above them about what they do?